File: porntend HHQ/Dashboard/Dashboard/reamoteApp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using System.IO;

namespace Dashboard
{
    class reamoteApp
    {
        public void createReamoteAppFile(string remoteAppName , string ipServer)
        {
            string path = @"reamoteapp.rdp";
            File.Delete(path);

            if (!File.Exists(path))
            {
                // Create a file to write to.
                using (StreamWriter sw = File.CreateText(path))
                {
                    sw.WriteLine("allow desktop composition:i:1");
                    sw.WriteLine("allow font smoothing:i:1");
                    sw.WriteLine("alternate full address:s:" + ipServer);
                    sw.WriteLine("alternate shell:s:rdpinit.exe");
                    sw.WriteLine("devicestoredirect:s:*");
                    sw.WriteLine("full address:s:" + ipServer);
                    sw.WriteLine("prompt for credentials on client:i:1");
                    sw.WriteLine("promptcredentialonce:i:0");
                    sw.WriteLine("redirectcomports:i:1");
                    sw.WriteLine("redirectdrives:i:1");
                    sw.WriteLine("remoteapplicationmode:i:1");
                    sw.WriteLine("RemoteProgram:s:" + remoteAppName);
                    sw.WriteLine("remoteapplicationprogram:s:||" + remoteAppName);
                    sw.WriteLine("span monitors:i:1");
                    sw.WriteLine("use multimon:i:1");
                }
            }
            System.Diagnostics.Process.Start(path);
            //File.Delete(path);
        }
    }
}

File: HouseHQ_web/HHQ_web/HHQ_web/remoteApp/reamoteapp.c
#include <stdio.h>
#include <string.h>
#include <windows.h>
#include <unistd.h>


int main ()
{
  char IP_server[] = "shaypc";
  char remoteAppName[] = "notepad++";
  char fileName[] = "remoteApp.rdp";
  FILE * fp;


  fp = fopen (fileName,"w");


  fprintf (fp, "allow desktop composition:i:1%c", 10);
  fprintf (fp, "allow font smoothing: i:1%c", 10);
  fprintf (fp, "alternate full address: s:%s%c", IP_server, 10);
  fprintf (fp, "alternate shell: s:rdpinit.exe%c", 10);
  fprintf (fp, "devicestoredirect: s:*%c", 10);
  fprintf (fp, "full address: s:%s%c", IP_server, 10);
  fprintf (fp, "prompt for credentials on client:i:1%c", 10);
  fprintf (fp, "promptcredentialonce: i:0%c", 10);
  fprintf (fp, "redirectcomports: i:1%c", 10);
  fprintf (fp, "redirectdrives: i:1%c", 10);
  fprintf (fp, "remoteapplicationmode: i:1%c", 10);
  fprintf (fp, "RemoteProgram: s:%s%c", remoteAppName, 10);
  fprintf (fp, "remoteapplicationprogram: s:||%s%c", remoteAppName, 10);
  fprintf (fp, "span monitors: i:1%c", 10);
  fprintf (fp, "use multimon: i:1%c", 10);


  fclose (fp);


  system(fileName);
  Sleep(10);
  do{remove(fileName);}while(access( fileName, F_OK ) == 0);


  return 0;
}

